addons/mrp/models/mrp_workorder.py

```diff
@@ -32,7 +32,7 @@ def _read_group_workcenter_id(self, workcenters, domain, order):
         string='Workcenter Status', related='workcenter_id.working_state') # technical: used in views only
     product_id = fields.Many2one(related='production_id.product_id', readonly=True, store=True, check_company=True)
     product_tracking = fields.Selection(related="product_id.tracking")
-    product_uom_id = fields.Many2one('uom.uom', 'Unit of Measure', required=True, readonly=True)
```

Comment:
we may be able to completely get rid of the `mrp_workorder.product_uom_id` (or replace it with a related field) at this point since it doesn't look like it should ever be different from `production_id.product_uom_id` and we rarely reference `mrp_workorder.product_uom_id` directly (we directly reference the `production_id.product_uom_id` one instead).